dotcom-rendering/cypress/e2e/parallel-4/show-more-test.cy.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-undef */
+/* eslint-disable func-names */
+import { disableCMP } from '../../lib/disableCMP';
+import { setLocalBaseUrl } from '../../lib/setLocalBaseUrl.js';
+
+import showMoreRequestBody from '../../fixtures/manual/science-show-more-req.json';
+import scienceFrontJson from '../../fixtures/manual/science-front.json';
```

Comment:
In DCR we typically don't use default exports; we favour named exports. It helps because it's then easier to trace the name of the file through the code (it doesn't change). We also keep the name of the file the same as the named export (for similar reasons!). That said it is true that there are many examples in the code of this pattern not being used!

Reply:
This is importing from the fixtures directory used by Storybook, Cypress has it's own fixtures folder

Reply:
> This is importing from the fixtures directory used by Storybook, Cypress has it's own fixtures folder

is it? this is the path from repo root on my computer: `dotcom-rendering/cypress/fixtures/manual/science-front.json`

Reply:
> In DCR we typically don't use default exports; we favour named exports.

ah okay, my reason for not using a named export here is because it's importing from a `.json` file, but if I understand you correctly then it would make sense to refactor so that it's importing from a `js` file instead?

Reply:
> is it? this is the path from repo root on my computer: `dotcom-rendering/cypress/fixtures/manual/science-front.json`

Oh, you're right. Sorry, I completely misread that!